packages/reports/addon/components/navi-table-select.js

```diff
@@ -22,5 +23,10 @@ export default Component.extend({
   /**
    * @property {Boolean} searchEnabled
    */
-  searchEnabled: false
+  searchEnabled: false,
```

Comment:
Can we change this component to be a native class?

Reply:
what do we mean by native class?

Reply:
Line 18. `export default Class NaviTableSelect extends Component {`

Reply:
```js
import { layout as templateLayout, tagName } from '@ember-decorators/component';

@templateLayout(layout)
@tagName('')
class NativeTableSelectComponent extends Component {
  /**
   * @property {Boolean} searchEnabled
   */
  searchEnabled = false;

  /**
   * @property {String} searchField
   */
  searchField = 'longName';
}

export default NativeTableSelectComponent;
```

Then in the template
```hbs
<div class="navi-table-select" ...attributes>
  {{!-- rest of template here --}}
</div>
```

Reply:
ohhh I got what you mean now